### PR TITLE
Popup Window

### PR DESCRIPTION
**Pull request by m00h0**
- Finish the Html flie.
- Finish the CSS file.
- Finish the Readme.md file.
- Add the Menu bar.
- Add the Popup window.
- There issue with Linter can't be solved you are aware of it.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -96,11 +96,11 @@ <h1>
...
       </div>
     </section>
-    <section class="projects" id="my-projects">
+    <div class="projects">
       <h2>My Recent Works</h2>
       <span class="line"></span>
-      <div class="project-cards">
-        <div class="project-card">
+       <div class="project-cards" id="my-portfolio">
+        <!-- <div class="project-card">
```
Comment (christianbiring1) [inline index.html]: - [ ] As mentioned in the project requirements, kindly make sure your project section is rendered dynamically through javascript. so make sure you delete it here in your markup and render it with javascript. As i can see you have commented this section, please make sure you don;t have the commented lines of code in your code base. you may consider dealing them

File: script.js [hunk 2]
```diff
@@ -10,3 +10,215 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
...
+    supdescription: 'Keeping track of hundreds of components',
+    projectInfo: { company: 'Microverse', role: 'Back END Dev', year: 2015 },
+    seeLive: 'https://github.com/newtonah0/Mobile-Portfolio',
+    seeSource: 'https://github.com/newtonah0/Mobile-Portfolio',
+  },
+
+  {
+    id: 6,
+    title: 'Multi-Post Stories Gain+Glory',
+    snapshot: 'Images/assets/Snapshoot-Portfolio.svg',
```
Comment (christianbiring1) [inline script.js]: - [ ] For a better user experience, Kindly make sure you are using the correct path on your images, as you may notice how your images are broken.
image

File: script.js [hunk 3]
```diff
@@ -10,3 +10,215 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
...
+  bodySection.append(section);
+});
+const body = document.querySelector('section');
+projects.forEach((project) => {
+  const popupWindow = document.createElement('div');
+  popupWindow.className = 'popup-window';
+  popupWindow.id = `card-${project.id}`;
+  popupWindow.innerHTML = `
+  <section class="project-card">
+  <div class="pop-header">
```
Comment (christianbiring1) [inline script.js:122]: - [ ] Kindly make sure you stick to the original template as much as possible. Your typos (fonts size, weight), color, spacing are supposed to be similar with the original design.
As you may notice the difference between your popup modal and the original design 
### Original design :point_down: 
image
### Your design :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @newtonah0 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 


## Highlights
- [ ] Great work on implementing the popup modal as required in the project description:+1: 
- [ ] All linters checks are passing :+1: 
- [ ] Descriptive PR `title` and `summary` :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 3 above)
Comment (newtonah0) [reply script.js:122]: image
my design and my images is work on my browser?

Comment (newtonah0) [reply script.js:122]: image
and that in the desktop version.

Comment (m00h0) [reply script.js:122]: image
it's show in my design? @christianbiring1

File: script.js [hunk 4]
```diff
@@ -10,3 +10,215 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
...
+  <div class="action-btn-container">
+  <button class="green-button" type="button" id="btn-${project.id}" name="button">See project</button>
+  
+  </div>
+</div>
+`;
+
+  bodySection.append(section);
+});
+const body = document.querySelector('section');
```
Comment (codecaiine) [inline script.js:115]: - [ ] Good job so far by implementing some changes requested by the previous reviewer But I think there are a few things you need to fix in your design like different element positions (Buttons.. etc). Please make sure to stick to the Figma-chosen design as much as possible. You can check the element below to see the differences ⬇️ 

##### Figma design
windows


##### Video with your design
https://www.loom.com/share/18c4eb0fd4c0420faf7a7ec750939443?sid=e93334e3-9f63-4a0a-9f82-05ca7ef634b5

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @newtonah0 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Good request changes application ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by codecaiine** [comment]
Hi @newtonah0 , I noticed that you pushed some code while I was submitting your project review. Please note that this creates confusion. Please always double-check the project requirement or the reviewers before submitting your project.

**Review by codecaiine — APPROVED** [review]
Hi @newtonah0 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good request changes application ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**